Week10/Week10.playground/Contents.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+// Struct for decoding JSON response
+struct CatFactResponse: Codable {
+    let fact: String
+}
+
+// Struct for AsyncSequence
+struct CatFactSequence: AsyncSequence {
+    typealias Element = CatFactResponse
+    
+    // Iterator for CatFactSequence
+    struct Iterator: AsyncIteratorProtocol {
+        let count: Int
+        var currentCount = 0
```

Comment:
You can omit these two and just have `getCatFacts` to run the loop for `count` which is what you already have in the code